travis/run-large-python-tests.sh

```diff
@@ -37,8 +35,11 @@ pytest --verbose tests/tensorflow/test_tensorflow_model_export.py --large
 pytest --verbose tests/tensorflow_autolog/test_tensorflow_autolog.py --large
 pytest --verbose tests/azureml --large
 pytest --verbose tests/models --large
+pytest --verbose tests/keras --large
 pytest --verbose tests/keras_autolog --large
 pip install 'tensorflow>=2.0.0'
 pytest --verbose tests/tensorflow/test_tensorflow2_model_export.py --large
 pytest --verbose tests/tensorflow_autolog/test_tensorflow2_autolog.py --large
+pip install 'keras>=2.3.1'
```

Comment:
Hm I guess this gives us a bit more coverage (to install old Keras first & run tests, then install the new Keras) - but I think it's fine to just test against the new Keras (e.g. just update https://github.com/mlflow/mlflow/blob/master/travis/large-requirements.txt#L27 to Keras 2.3.1). We should in a separate follow-up find a long-term fix for correctly capturing which dependencies the user installed via conda vs pip, and recording that info in the conda.yaml saved with the model (would you mind filing a GitHub issue to track that work)?

Reply:
I can make the GitHub issue! In regards to changing the default Keras testing version, a keras_autolog test fails due to one of the metric's name being changed from `'acc'` to `'accuracy'` in Keras 2.3.1. I can go ahead and make the quick version check as part of this PR so we don't have to change the `run-large-python-tests.sh` twice unnecessarily.